test/test_integration.py

```diff
@@ -30,28 +33,53 @@ def tearDown(self):
     def test_dockstore_md5sum(self):
         """Fetch the md5sum cwl from dockstore, run it on the wes-service server, and check for the correct output."""
         cwl_dockstore_url = 'https://dockstore.org:8443/api/ga4gh/v2/tools/quay.io%2Fbriandoconnor%2Fdockstore-tool-md5sum/versions/master/plain-CWL/descriptor/%2FDockstore.cwl'
-        output_filepath = run_md5sum(cwl_input=cwl_dockstore_url)
+        output_filepath , _ = run_md5sum(cwl_input=cwl_dockstore_url)
 
         self.assertTrue(check_for_file(output_filepath), 'Output file was not found: ' + str(output_filepath))
         shutil.rmtree('workflows')
 
     def test_local_md5sum(self):
         """Pass a local md5sum cwl to the wes-service server, and check for the correct output."""
         cwl_local_path = os.path.abspath('testdata/md5sum.cwl')
-        output_filepath = run_md5sum(cwl_input='file://' + cwl_local_path)
+        output_filepath, _ = run_md5sum(cwl_input='file://' + cwl_local_path)
 
         self.assertTrue(check_for_file(output_filepath), 'Output file was not found: ' + str(output_filepath))
         shutil.rmtree('workflows')
 
+    def test_multipart_upload(self):
+        """Pass a local md5sum cwl to the wes-service server, and check for uploaded file in service."""
+        cwl_local_path = os.path.abspath('testdata/md5sum.cwl')
+        output_filepath = run_md5sum(cwl_input='file://' + cwl_local_path)
```

Comment:
Instead of:
`output_filepath = run_md5sum(cwl_input='file://' + cwl_local_path)` 
`_, run_id = output_filepath`
here, you can just write:
`_, run_id = run_md5sum(cwl_input='file://' + cwl_local_path)`